test/test_parmed_structure_slicing.py

```diff
@@ -238,6 +239,46 @@ def test_chain_residue_atom_selection(self):
         self.assertRaises(ValueError, lambda:
                 pdb2[list(range(len(pdb2.atoms)+1))])
         self.assertRaises(ValueError, lambda: pdb2[[0,len(pdb2.atoms)]])
+        
+    def test_structure_box_and_space_group_and_symmetry(self):
+        def assert_correctly_copy(parm):
+            sliced_parm = parm['@1-3']
+            np.testing.assert_equal(parm.box, sliced_parm.box)
```

Comment:
parm.box and sliced_parm.box can both be None (and are for several files, like the PSF which has no unit cell information)